convert/jenkinsjson/convert.go

```diff
@@ -62,7 +62,7 @@ type ProcessedTools struct {
 var mavenGoals string
 var gradleGoals string
 
-var defaultImage string
+var defaultWindowsImage string = "mcr.microsoft.com/powershell"
```

Comment:
just to keep in mind: pwsh and powershell aren't the same as powershell.exe is windows specific and pwsh (powershell core) is cross platform ([ref](https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.4)) and earlier when I worked on this, I think this image only contains pwsh and not powershell.exe